packages/vm/src/features.rs

```diff
@@ -0,0 +1,77 @@
+use parity_wasm::elements::{Internal, Module};
+use std::collections::HashSet;
+use std::iter::FromIterator;
+use wasmer_runtime_core::{export::Export, instance::Instance};
+
+static REQUIRES_PREFIX: &str = "requires_";
+
+pub fn required_features_from_wasmer_instance(wasmer_instance: &Instance) -> HashSet<String> {
```

Comment:
very nice with both implementations.

Reply:
but is this function tested anywhere?
I only noticed `required_features_from_module` being used.

Reply:
It is tested indirectly in the Instance, where it is used in `Instance::from_wasmer`. Testing it here would probably be complicated, since you need to create an instance and then `wasmer_instance` is not public.

Reply:
Yeah, I saw that a bit later, and how it was exposed (in the next comment).
Nice work on that design